scripts/syslog-notify-oxidized.php

```diff
@@ -1,47 +1,37 @@
 #!/usr/bin/env php
 <?php
 
-$init_modules = array();
+$init_modules = [];
 require __DIR__ . '/../includes/init.php';
 
-function oxidized_node_update($hostname, $username, $msg)
+function oxidized_node_update($hostname, $msg, $username = 'not_provided')
 {
     global $config;
 
     // Work around https://github.com/rack/rack/issues/337
     $msg = str_replace("%", "", $msg);
-    $postdata = array("user" => $username, "msg" => $msg);
-
-    $version = `git rev-parse --short HEAD`;
-    if ($version != "") {
-        $version = "/".$version;
-    }
+    $postdata = ["user" => $username, "msg" => $msg];
 
     $curl = curl_init();
```

Comment:
@laf I noticed we pull in http://requests.ryanmccue.info/ but it is only used for peeringdb.
Thoughts on using this all over instead of curl?

Reply:
Good shout, updated + used `Config::get()`